packages/core/src/utils/zod.ts

```diff
@@ -4,17 +4,45 @@ import { OpenAPIV3 } from 'openapi-types';
 import {
   ZodArray,
   ZodBoolean,
+  ZodLiteral,
   ZodNativeEnum,
   ZodNullable,
   ZodNumber,
   ZodObject,
   ZodOptional,
   ZodString,
+  ZodType,
+  ZodUnion,
   ZodUnknown,
 } from 'zod';
 
 import RequestError from '@/errors/RequestError';
 
+const zodLiteralToSwagger = (zodLiteral: ZodLiteral<unknown>): OpenAPIV3.SchemaObject => {
+  const { value } = zodLiteral;
+
+  switch (typeof value) {
+    case 'boolean':
+      return {
+        type: 'boolean',
+        format: String(value),
+      };
+    case 'bigint':
```

Comment:
![image](https://user-images.githubusercontent.com/36393111/173976661-fdf67251-df2d-47e0-9e5d-a973a30afd92.png)

Reply:
Updated to:

```ts
    case 'string':
      return {
        type: 'string',
        format: value === '' ? 'empty' : `"${value}"`,
      };
```